### PR TITLE
Add guidance on the AUTHORS file to contributors

### PR DESCRIPTION
**Pull request by mehrdada**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,12 +39,18 @@ How to get your contributions merged smoothly and quickly.
 - Don't fix code style and formatting unless you are already changing that line to address an issue. PRs with irrelevant changes won't be merged. If you do want to fix formatting or style, do that in a separate PR.
  
 - Unless your PR is trivial, you should expect there will be reviewer comments that you'll need to address before merging. We expect you to be reasonably responsive to those comments, otherwise the PR will be closed after 2-3 weeks of inactivity.
+
+- If you have non-trivial contributions, please consider adding an entry to the AUTHORS file
+  listing the original copyright holder (yourself, if you are signing the
+  individual CLA, or your company, for corporate CLAs) in the same PR as your
+  contribution.  Obviously this needs to be done only once, for each company, or
```
Comment (hsaliak) [inline CONTRIBUTING.md]: s/Obviously//

**Comment by hsaliak** [comment]
looks good, just a minor nit.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,12 +39,19 @@ How to get your contributions merged smoothly and quickly.
 - Don't fix code style and formatting unless you are already changing that line to address an issue. PRs with irrelevant changes won't be merged. If you do want to fix formatting or style, do that in a separate PR.
  
 - Unless your PR is trivial, you should expect there will be reviewer comments that you'll need to address before merging. We expect you to be reasonably responsive to those comments, otherwise the PR will be closed after 2-3 weeks of inactivity.
+
+- If you have non-trivial contributions, please consider adding an entry to [the
+  AUTHORS file](https://github.com/grpc/grpc/blob/master/AUTHORS) listing the
+  copyright holder for the contribution (yourself, if you are signing the
+  individual CLA, or your company, for corporate CLAs) in the same PR as your
+  contribution.  This needs to be done only once, for each company, or
```
Comment (hsaliak) [inline CONTRIBUTING.md]: This sentence begins with 2 spaces, which is inconsistent with the rest of the file. Could you please remove 1 space?

Comment (mehrdada) [reply CONTRIBUTING.md]: No. This is Markdown. :)

**Pull request merged**